### DISCUSSION TITLE
Azure monitor: support for container apps metrics

### DISCUSSION THREAD

**Discussion by despian** [grafana/grafana#52014] [Data Sources]
It's not currently possible to access metrics for Azure Container Apps ("namespace": "Microsoft.App/containerApps").

Resources of this type do not appear in the resource picker for the Azure Monitor data source.

I'd be happy to help add support for this resource type if someone could point me in the right direction.

Edit:

Seems this is related to outdated metadata in the UI. I have created Issue #52028 and a related PR #52030

**Comment by andresmgot** [comment]
Thanks @despian, since the PR has been already merged let me close this discussion. Thanks again for the contribution!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
